src/doc/src/reference/profiles.md

```diff
@@ -113,11 +113,12 @@ strip either symbols or debuginfo from a binary. This can be enabled like so:
 # ...
 
 [profile.release]
-strip = "debuginfo"
+strip = "symbols"
 ```
 
 Possible string values of `strip` are `"none"`, `"debuginfo"`, and `"symbols"`.
-The default is `"none"`.
+For debug profiles the default is `"none"`, and for profiles with `debug = false` 
+the default is `"debuginfo"`.
 
```

Comment:
> [Rust 1.77](https://blog.rust-lang.org/2024/03/21/Rust-1.77.0.html#enable-strip-in-release-profiles-by-default) changed the default behaviour on debug symbols stripping. That change seems not to be yet updated to the book, which this PR fixes.

I wonder if we need to update the documentation.  The idea behind this is to emulate `std` respecting `debug`.